mmtrack/models/losses/triplet_loss.py

```diff
@@ -0,0 +1,64 @@
+import torch
+import torch.nn as nn
+from mmdet.models import LOSSES
+
+
+@LOSSES.register_module()
+class TripletLoss(nn.Module):
+    """Triplet loss with hard positive/negative mining.
+
+    Reference:
+        Hermans et al. In Defense of the Triplet Loss for
+            Person Re-Identification. arXiv:1703.07737.
+    Imported from `<https://github.com/KaiyangZhou/deep-person-reid/blob/
+        master/torchreid/losses/hard_mine_triplet_loss.py>`_.
+    Args:
+        margin (float, optional): margin for triplet. Default is 0.3.
```

Comment:
1. The styles of the captions should be consistent. You can use `Default: ***`, or `default to **`
2. The descriptions should be completed words, at least it should be `margin for triplet loss` here